Trading/Mode/grid_trading_mode/grid_trading.py

```diff
@@ -206,6 +224,8 @@ def get_default_pair_config(self, symbol, flat_spread, flat_increment) -> dict:
           self.CONFIG_USE_FIXED_VOLUMES_FOR_MIRROR_ORDERS: False,
           self.CONFIG_USE_EXISTING_ORDERS_ONLY: False,
           self.CONFIG_ALLOW_FUNDS_REDISPATCH: False,
+          self.CONFIG_ENABLE_TRAILING_UP: False,
+          self.CONFIG_ENABLE_TRAILING_DOWN: False,
```

Comment:
Why False as default value?

Reply:
Do you think we should enable it by default ? I would say no but we create a default profile with a trailing grid instead

Reply:
That's a good idea (only for trailing UP I think)

Reply:
Yes, only trailing up, i'll add it